docs/source/recipes/plot_18_recipe.py

```diff
@@ -0,0 +1,140 @@
+"""
+Calculating the Pearson correlation coefficient between datasets
+================================================================
+
+In this recipe, we will take two datasets, one for an independent variable
+(in this example elevation) and one for a dependent variable (snow
+cover over a particuar day), regrid them to the same resolution then
+calculate the correlation coefficient, to get a measure of the relationship
+between them.
+
+"""
+
+# %%
+# 1. Import cf-python, cf-plot and other required packages:
+import cfplot as cfp
+import cf
+
+import matplotlib.pyplot as plt
+import scipy.stats.mstats as mstats
+
+# %%
+# 2. Read the data in and unpack the Fields from FieldLists using indexing.
+# In our example We are investigating the influence of the land height on
+# the snow cover extent, so snow cover is the dependent variable. The snow
+# cover data is the
+# 'Snow Cover Extent 2017-present (raster 500 m), Europe, daily – version 1'
+# sourced from the Copernicus Land Monitoring Service which is described at:
+# https://land.copernicus.eu/en/products/snow/snow-cover-extent-europe-v1-0-500m
+# and the elevation data is the 'NOAA NGDC GLOBE topo: elevation data' dataset
+# which can be sourced from the IRI Data Library, or details found, at:
+# http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NGDC/.GLOBE/.topo/index.html.
+orog = cf.read("~/recipes/1km_elevation.nc")[0]
+snow = cf.read("~/recipes/snowcover")[0]

```

Comment:
Both of these are new datasets we need to use, so I have uploaded them to the 'recipes' Google Drive folder in readiness.